source/routes/delete/session.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  '/delete/session': {
+    methods: ['delete'], // TODO: We may want to use post instead?

```

Comment:
Another TODO... maybe say "for a future story" rather than TODO? It ultimately means the same thing, I suppose.

Reply:
TODO is easier to find. Many text editors/IDEs provide features around 'TODO'. Using any other prefix would needlessly complicate things.